### PR TITLE
IE keyboard navigation fix

### PR DESCRIPTION
**Pull request by kryswisnaskas**
**Description of change**
Redirects focus to the date input field after selecting a date from the calendar. This allows the keyboard navigation to continue in IE. 

**How to test**
- pull changes
- create a report in IE
- select a date
- use keyboard navigation
- verify that navigation moves forward and doesn't skip to the top

Alternatively use Chrome to test:
- pull changes
- create a report
- select a date
- verify that after date selection, the input date has focus

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/179

**Checklist**

- [x] Meets issue criteria
- [x] Code tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] Documentation updated
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified

### CODE REVIEW

File: frontend/src/components/DatePicker.js [hunk 1]
```diff
@@ -61,6 +61,8 @@ const DateInput = ({
                 onDateChange={(d) => {
                   const newDate = d ? d.format(dateFmt) : d;
                   onChange(newDate);
+                  const input = document.getElementById(name);
+                  if (input) input.focus();
```
Comment (kryswisnaskas) [inline frontend/src/components/DatePicker.js:65]: Ideally, we would want to use `ref `to switch focus, however, I run into problems with that approach.

**Review by dcmcand — APPROVED** [review]
I could only test in chrome, but it worked for me. After choosing a date, focus was back to the date input

**Review by jasalisbury — APPROVED** [review]
:rocket: works for me on IE and chrome

**Comment by gopar** [comment]
Same, works on IE \o/

**Review by rahearn — APPROVED** [review]

**Pull request merged**